### PR TITLE
Keep only external links checks

### PR DESCRIPTION
**Pull request by veggiemonk**
fixes #908

### CODE REVIEW

**Review by agebhar1 — APPROVED** [review]
LGTM :+1:

**Comment by mash1t** [comment]
@veggiemonk @agebhar1 

The last two commits contain the following changes:
- move tests to subfolder ./tests
- remove some not needed code in tests/pull_request.js
- add tests/all.js for running tests with GitHub token locally via `npm run test`
- move duplicate methods to helper.js

It would be great if you could give me feedback, especially on the naming of the files.

File: tests/pull_request.js [hunk 1]
```diff
@@ -144,6 +137,8 @@ async function main() {
 
   console.log(`checking ${github_links.length} GitHub repositories...`);
 
+  console.log(`skipping GitHub repository check. Run "npm run test" to execute them manually.`);
```
Comment (agebhar1) [inline tests/pull_request.js:50]: the 'test' script was removed from package.json within this commit

**Review by agebhar1 — CHANGES_REQUESTED** [review]

**Review by agebhar1 — APPROVED** [review]
I'm fine with the changes :+1:

**Comment by veggiemonk** [comment]
I'll have a look 😄

**Comment by veggiemonk** [comment]
Thank you @mash1t and @agebhar1 

Nice work 😄

**Pull request merged**